Dockerfiles/openxt-buster-oe64

```diff
@@ -50,6 +50,10 @@ RUN ln -s /lib64/ld-linux-x86-64.so.2 /lib/
 
 RUN useradd -Ums /bin/bash -l -p '""' -u $UID $UNAME
 
+# Add quilt config files
+ADD root/home/build/.quiltrc /home/build/.quiltrc
+ADD root/home/build/openxt/oxt-patch.header /home/build/openxt/oxt-patch.header
```

Comment:
Docker mounts $TOP, which is the openxt manifest directory, over /home/build/openxt: https://github.com/OpenXT/bordel/blob/d9c52e1bab6c7e5a36241df24838e66ce79579f4/cmds/docker#L60 Does oxt-patch.header still show inside the container?

Reply:
Ah you're right, my testing was flawed. Misused the docker command. I'm just going to put it in the home dir in that case.

Reply:
Ok, sounds good.